scala/bigtable-shc/README.md

```diff
@@ -71,45 +89,47 @@ Create environment variables for the following commands:
     BIGTABLE_TABLE=my-table
     SPARK_CLUSTER=spark-cluster
 
-## Build the jar
+## Building (Assembling) Example
 
-The Spark job is assembled into a uber/fat jar with all of its dependencies and hbase configuration. To build, run:
+The Spark application is assembled into an uber/fat jar with all of its dependencies and hbase configuration. To build, run:
 
-    mvn assembly:assembly
+    mvn clean assembly:assembly
 
-Note that since the Bigtable configuration is included in the fat jar, any changes
+The above command should build `target/cloud-bigtable-dataproc-spark-shc-0.1-jar-with-dependencies.jar` file.
+
+**NOTE**: Since the Bigtable configuration is included in the fat jar, any changes
  will require a repackaging of the uberjar.
 
-## Test your job locally (optional but recommended)
+## Testing Locally (Optional but Recommended)
 
-This step requires a local Spark installation.
+This step requires a local installation of Apache Spark.
 
-While you will need a real Bigtable cluster, you can test the Spark job locally,
-if you have Spark insatlled. For testing, consider a Bigtable development
+While you need a real Bigtable cluster, you can test the sample locally,
+if you have Spark installed. For testing, consider a Bigtable development
 cluster.
 
-    spark-submit --class com.example.bigtable.spark.shc.BigtableSource --master local[8] \
-      target/cloud-bigtable-dataproc-spark-shc-0.1-jar-with-dependencies.jar $BIGTABLE_TABLE
+    $SPARK_HOME/bin/spark-submit \
+      --class com.example.bigtable.spark.shc.BigtableSource \
+      target/cloud-bigtable-dataproc-spark-shc-0.1-jar-with-dependencies.jar \
+      $BIGTABLE_TABLE
 
+**NOTE**: `BIGTABLE_TABLE` is one of the environment variables defined when [configuring the sample](#configuring-the-sample).
 
-### Submit the job to Cloud Dataproc
+### Submitting Example to Google Cloud Dataproc
```

Comment:
Old one seemed to be better too?
"Submit Job to Cloud Dataproc"

Reply:
Jobs were in Hadoop MapReduce while in Spark they are applications (that have one or more jobs). I wanted to make a distinction between the terms, but changed it per comment.